src/utility.js

```diff
@@ -0,0 +1,32 @@
+  function getSortedData(productList, sortBy) {
+    if (sortBy === "HIGH_TO_LOW")
+      return productList.sort((a, b) => b.price - a.price);
+    if (sortBy === "LOW_TO_HIGH")
+      return productList.sort((a, b) => a.price - b.price);
```

Comment:
Suggestion: `sort` function mutate array and in Pure programming we didn't mutate parameters also, you can make copy of array here and then sort it

Reply:
Done!